### ISSUE TITLE
New mcmc chain lengths break build() code

### ISSUE THREAD

**Issue by cgrandin** [cgrandin/hake-assessment#351]
I'll fix the code to work with differing numbers of samples.

**Comment by taylori** [comment]
I'm seeing this error in `build(T,T,T)`:

```
run.extra.mcmc.models: Running model for each mcmc sample to get additional mcmc outputs...


Assuming version 3.30 based on number of numeric values.
overwriting file: ../../models/2018.40_base_model/extra-mcmc/starter.ss 
Error in `[<-.data.frame`(`*tmp*`, newpar$label %in% names(posts), 1,  : 
  replacement has 225 rows, data has 223
```
This might be a problem caused by change in the count of parameters for creating the .par files from the rows in the posteriors.sso file. Or perhaps something with parameters that have parentheses in their labels, like the D-M parameters "ln(EffN_mult)_1", except that this wasn't an issue with other parameters like that.

You've got a head start, so I won't try to dig deeper unless you want me to help.

**Comment by cgrandin** [comment]
Yes, that's what it is. I've isolated the code, it's in the `run.extra.mcmc.models` function. I'll get it going.

**Comment by cgrandin** [comment]
Fixed in commit 85f69ca16f0535877a998cfba4093875a578d1e3
